agent/base

```diff
@@ -278,12 +251,12 @@ function generate_inventory {
     printf -- ${preamble} "${hostname}"
     for dirent in $(/bin/ls -1 ${tool_group_dir}); do
         if [[ "${dirent}" == "__trigger__" ]]; then
-		# Ignore trigger files
-		continue
-	elif [[ ! -d ${tool_group_dir}/${dirent} ]]; then
-		# Ignore spurious files
-		continue
-	fi
+            # Ignore trigger files
```

Comment:
Does the FIXME on l.242 need to be addressed?

Reply:
Thanks, removed the comment at line 242.